src/packages/client/src/__tests__/runtime-tests/disconnect-finally/index.js

```diff
@@ -0,0 +1,21 @@
+const { PrismaClient } = require('@prisma/client')
+const assert = require('assert')
+
+let res
+const prisma = new PrismaClient()
+async function main() {
+  res = prisma.user.findMany()
+}
+
+module.exports = () => {
+  return new Promise((r) => {
+    main().finally(async () => {
+      await prisma.$disconnect()
+      r(await res)
```

Comment:
for readability resolve would be better, I was surprised to see this `r` 😄 looked like a typo at first sight

Reply:
Good point, will rename.